docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,462 @@
+| wishlist | [{{wishlist}}](/theme-objects/wishlist) |
+
+
+A standard `BODL` instance contains the following calculated properties:
+
+| BODL Property | First-Level Child Properties | 
+|:---|:---|
+| category: object | name: string, products: array |
+| product: object | id: string, title: string, price: object |
+
+A standard `BODL` instance contains the following methods:
+
+| BODL Method | Parameter(s) | Parameter Properties | Return Value |
+|:---|:---|:---|:---|
+| getCartItemContentId | `item`: object | `item.type`: string, `item.product_id`: string | If the item is a gift certificate, returns the item's `type`. Otherwise, returns the item's `product_id`.  |
+| getQueryParamValue | `name`: string | **X** | Returns the value of the specified query parameter from the window's URL. |
+
+## Initialize BODL
+
+The following script extracts storefront data from the Stencil objects available in the front-end environment to construct a standard `BODL` object.
+
+```handlebars title="Sample Script Code Start: Initialization Script & Page Event" lineNumbers
+<script>
+  
+  if (typeof BODL === 'undefined') {
+    // https://developer.bigcommerce.com/theme-objects/breadcrumbs
+    {{inject "breadcrumbs" breadcrumbs}}
+    // https://developer.bigcommerce.com/theme-objects/brand
+    {{inject "brand" brand}}
+    // https://developer.bigcommerce.com/theme-objects/category
+    {{inject "categoryProducts" category.products}}
+    {{inject "categoryName" category.name}}
+    {{inject "customerId" customer.id}}
+    {{inject "customerEmail" customer.email}}
+    {{inject "customerPhone" customer.phone}}
+    // https://developer.bigcommerce.com/theme-objects/product
+    {{inject "productId" product.id}}
+    {{inject "productTitle" product.title}}
+    {{inject "productCurrency" product.price.without_tax.currency}}
+    {{inject "productPrice" product.price.without_tax.value}}
+    // https://developer.bigcommerce.com/theme-objects/products
+    {{inject "products" products}}
+    // https://developer.bigcommerce.com/theme-objects/product_results
+    {{inject "search" product_results}}
+    // https://developer.bigcommerce.com/theme-objects/order
+    {{inject "order" order}}
+    // https://developer.bigcommerce.com/theme-objects/wishlist
+    {{inject "wishlist" wishlist}}
+    // https://developer.bigcommerce.com/theme-objects/cart_id
+    {{inject "cartId" cart_id}}
+    // https://developer.bigcommerce.com/theme-objects/cart
+    // (Fetching selective cart data to prevent additional payment button object html from causing JS parse error)
+    {{inject "cartItemAdded" cart.added_item}}
+  }
+  var BODL = JSON.parse({{jsContext}});
+
+  if (BODL.categoryName) {
+  
+      BODL.customer = {
+        id: BODL.customerId,
+        email: BODL.customerEmail,
+        phone: BODL.customerPhone
+      }
+    }
+
+    BODL.category = {
+      name: BODL.categoryName,
+      products: BODL.categoryProducts,
+    }
+  }
+  if (BODL.customerId) {
```

Comment:
This check looks out of place now? It should have the BODL.customer object creation within it. 

So the whole BODL.categoryName to BODL.productTitle block here should be like:
```
if (BODL.categoryName) {
  BODL.category = {
    name: BODL.categoryName,
    products: BODL.categoryProducts,
  } 
}

if (BODL.customerId) {
  BODL.customer = {
    id: BODL.customerId,
    email: BODL.customerEmail,
    phone: BODL.customerPhone
  }
}

if (BODL.productTitle) {
  BODL.product = {
    id: BODL.productId,
    title: BODL.productTitle,
    price: {
      without_tax: {
        currency: BODL.productCurrency,
        value: BODL.productPrice,
      },
    },
  }
}
```